src/ui/popup.js

```diff
@@ -70,11 +71,11 @@ class Popup extends Evented {
      */
     addTo(map) {
         this._map = map;
-        this._map.on('move', this._update);
+        this._map.on('move', this._onMove);
         if (this.options.closeOnClick) {
             this._map.on('click', this._onClickClose);
         }
-        this._update();
+        this._update(false);
```

Comment:
The problem with this is a marker added within -180 to 180 long might not be shown on the map initially.

eg. in this view australia is world -1 and I've added a marker at long 170 but it doesn't show up until I move the map.
![selection_561](https://cloud.githubusercontent.com/assets/117278/25262005/9941e358-2698-11e7-89db-ef5982c45123.png)